### PR TITLE
feat: adds logic and function for user submit

### LINKED ISSUE #131: Create Users submissions table
This will require someone with Supabase access to create the table and migration. 

## Details

- [x] Create a table for user submissions
- [ ] Create a database.js funciton to upvote the submitted repo
- [ ] Check if the repo already exist in the DB before adding. 

## Question
Should the user still be associated to the submission?

We will need to wait on https://github.com/open-sauced/hot/pull/122

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds logic and supabase function to add user submissions. These user submissions will allow folks to add repos to the lists for consideration. 

## Related Tickets & Documents

Fixes #131 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, need vitest
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [x] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

Need to implement the UI #132 

npm run db

### CODE REVIEW

**Comment by 0-vortex** [comment]
We will need to architect the database changes as migrations for this, also didn't see it's still a draft :D

**Comment by bdougie** [comment]
Needs a rebase but also
- [ ] hover card color does match 
- [ ] need logic to discover last contribution time

Screen Shot 2022-04-13 at 4 05 35 PM

File: src/components/PostsWrap.jsx [hunk 1]
```diff
@@ -28,6 +28,11 @@ const PostsWrap = () => {
     setLimit((prevLimit) => prevLimit + 25);
   };
 
+  // const handleSubmitRecommendation = (userId, repo) => {
```
Comment (NsdHSO) [inline src/components/PostsWrap.jsx:31]: This line of code did not exists for us, is this a intended change?

**Comment by bdougie** [comment]
This has diverged too much and will need to integrate the new api instead.

**Pull request closed without merging**